pytest-embedded-idf/pytest_embedded_idf/unity_tester.py

```diff
@@ -685,6 +882,15 @@ def run_case(
             timeout: timeout in second
             start_retry (int): number of retries for a single case when it is failed to start
         """
+        warnings.warn(
+            'Current timeout logic not clear: currently it is a sum of wait board menu time and runtest time.'
+            'Will be renamed and split into two time in future',
+            DeprecationWarning,
+        )
```

Comment:
let's remove this warning.

`warnings.warn` should tell what the developer should do. let's update the docstring to

```python
        Args:
            case: the specific case that parsed in test menu
            reset: whether to perform a hardware reset before running a case
            timeout: timeout in second, setup time excluded  # <-- change here
            start_retry (int): number of retries for a single case when it is failed to start
```